test/framework/options.py

```diff
@@ -5088,20 +5089,21 @@ def test_github_new_pr_delete(self):
             print("Skipping test_new_pr_delete, no GitHub token available?")
             return
 
+        ec_name = 'bzip2-1.0.8.eb'
         args = [
             '--new-pr',
             '--github-user=%s' % GITHUB_TEST_ACCOUNT,
-            ':bzip2-1.0.6.eb',
+            f':{ec_name}',
             '-D',
             '--disable-cleanup-tmpdir',
-            '--pr-title=delete bzip2-1.6.0',
-            '--pr-commit-msg="delete bzip2-1.6.0.eb"'
+            f'--pr-title=delete {ec_name}',
+            f'--pr-commit-msg="delete {ec_name}"'
         ]
         txt, _ = self._run_mock_eb(args, do_build=True, raise_error=True, testing=False)
 
         regexs = [
             r"^== fetching branch 'develop' from https://github.com/easybuilders/easybuild-easyconfigs.git...",
-            r'title: "delete bzip2-1.6.0"',
+            r'title: "delete {ec_name}"',
```

Comment:
Missed this should have been an f-string

Reply:
Oh, thanks, too much C&P. Weird it wasn't detected by CI before

Reply:
The CI tests for GitHub integration don't run for PRs (as there is no visible GitHub token)

Reply:
(I've started https://github.com/easybuilders/easybuild-framework/pull/4841 from a branch rather than a fork so I can test the PR in the full context)

Reply:
You can set up your own fork to run the full test suite, incl. GitHub tests, but you need to provide a token yourself via GitHub Actions "secrets"